examples/mms/tts/tutorial/MMS_TTS_Inference_Colab.ipynb

```diff
@@ -0,0 +1,543 @@
+        },
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "'/content/vits'"
+            ],
+            "application/vnd.google.colaboratory.intrinsic+json": {
+              "type": "string"
+            }
+          },
+          "metadata": {},
+          "execution_count": 1
+        }
+      ],
+      "source": [
+        "%pwd\n",
+        "!git clone https://github.com/jaywalnut310/vits.git\n",
+        "!python --version\n",
+        "%cd vits/\n",
+        "\n",
+        "!pip install Cython==0.29.21\n",
+        "!pip install librosa==0.8.0\n",
+        "!pip install phonemizer==2.2.1\n",
+        "!pip install scipy\n",
+        "!pip install numpy\n",
+        "!pip install torch\n",
+        "!pip install torchvision\n",
+        "!pip install matplotlib\n",
+        "!pip install Unidecode==1.1.1\n",
+        "\n",
+        "%cd monotonic_align/\n",
+        "%mkdir monotonic_align\n",
+        "!python3 setup.py build_ext --inplace\n",
+        "%cd ../\n",
+        "%pwd"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KuBzieKbuJKN"
+      },
+      "source": [
+        "## 2. Choose a language and download its checkpoint\n",
+        "Find the ISO code for your target language [here](https://dl.fbaipublicfiles.com/mms/tts/all-tts-languages.html). You can find more details about the languages we currently support for TTS in this [table](https://dl.fbaipublicfiles.com/mms/misc/language_coverage_mms.html)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 2,
+      "metadata": {
+        "id": "UtEeQcmwuUaG",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "898560fb-4275-4c5d-d45d-2b40bddfa7b6"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Download model for language: eng\n",
+            "Model checkpoints in ./eng: ['G_100000.pth', 'config.json', 'vocab.txt']\n"
+          ]
+        }
+      ],
+      "source": [
+        "import os\n",
+        "import subprocess\n",
+        "import locale\n",
+        "locale.getpreferredencoding = lambda: \"UTF-8\"\n",
+        "\n",
+        "def download(lang, tgt_dir=\"./\"):\n",
+        "  lang_fn, lang_dir = os.path.join(tgt_dir, lang+'.tar.gz'), os.path.join(tgt_dir, lang)\n",
+        "  cmd = \";\".join([\n",
+        "        f\"wget https://dl.fbaipublicfiles.com/mms/tts/{lang}.tar.gz -O {lang_fn}\",\n",
+        "        f\"tar zxvf {lang_fn}\"\n",
+        "  ])\n",
+        "  print(f\"Download model for language: {lang}\")\n",
+        "  subprocess.check_output(cmd, shell=True)\n",
+        "  print(f\"Model checkpoints in {lang_dir}: {os.listdir(lang_dir)}\")\n",
+        "  return lang_dir\n",
+        "\n",
+        "lang = \"eng\"\n",
+        "ckpt_dir = download(lang)"
```

Comment:
```
lang = "eng"
```

Can you keep this at the top of the cell. Also, use capitals `LANG` - to make clear that users can pplay around with this.